### DISCUSSION TITLE
No requirement for sign in?

### DISCUSSION THREAD

**Discussion by BrakkF** [teslamate-org/teslamate#1459] [Q&A] [answered]
Hi,

Forgive me if this is a known issue/feature, I've searched through the discussions and couldn't find anything related. I'm running v1.22.0 in docker and I found that I don't need to log in from a remote location if I make my teslamate instance reachable via port forwarding when I'm already logged in somewhere else.

If I log off from any instance, then all instances will ask to sign in but as long as one instance in any location has signed in successfully, all other instances regardless of location are signed in as well.  Is this by design?  I don't see anywhere in the settings where I can set a timeout or something similar.  Can someone please point me in the right direction?  Thanks in advance.

J

**Comment by DrMichael** [comment, marked as answer]
Well, the solution here would be to use a reverse proxy as in https://docs.teslamate.org/docs/guides/apache or https://docs.teslamate.org/docs/guides/traefik.

**Reply by BrakkF** [reply]
Thanks, will try this later.
